internal/pkg/clock/testing.go

```diff
@@ -1,27 +1,30 @@
-package testhelpers
+package clock
 
 import (
 	"sync"
 	"time"
-
-	"github.com/filecoin-project/go-filecoin/internal/pkg/clock"
 )
 
-// FakeClock provides an interface for a clock which can be
-// manually advanced through time
+// Creates a new fake clock and chain clock wrapping it.
+func NewFakeChain(genesis uint64, epochDuration time.Duration, now int64) (Fake, ChainEpochClock) {
+	fake := NewFake(time.Unix(now, 0))
+	return fake, NewChainClockFromClock(genesis, epochDuration, fake)
+}
+
+// Fake provides an interface for a clock which can be manually advanced.
 // Adapted from: https://github.com/jonboulle/clockwork
-type FakeClock interface {
-	clock.Clock
-	// Advance advances the FakeClock to a new point in time, ensuring any existing
+type Fake interface {
+	Clock
+	// Advance advances the Fake to a new point in time, ensuring any existing
 	// sleepers are notified appropriately before returning
 	Advance(d time.Duration)
-	// BlockUntil will block until the FakeClock has the given number of
+	// BlockUntil will block until the Fake has the given number of
 	// sleepers (callers of Sleep or After)
 	BlockUntil(n int)
 }
 
-// NewFakeClock returns a FakeClock initialised at the given time.Time.
-func NewFakeClock(n time.Time) FakeClock {
+// NewFakeClock returns a Fake initialised at the given time.Time.
```

Comment:
nit: comment mismatch

Reply:
It would be nice if we could configure the linter to force consistency between function name and first comment word without forcing there to be a comment for all functions.

Reply:
The Go recommendation to redundantly start comments with the symbol being commented is silly. My preferred solution is to ignore this recommendation and write comments that won't rot.